go/1.18/libexec/src/net/http/h2_bundle.go:7130:48: ErrDeadlineExceeded not declared by package os

A small program fails to compile with tinygo dev when go 1.18 is installed.  Log:
```
$ go version
go version go1.18 darwin/amd64
$ env GOROOT=/usr/local/Cellar/go/1.18/libexec tinygo build -target=wasi -wasm-abi=generic -gc=leaking -scheduler=asyncify  -o bin/tinygo-example.wasm main.go
# net/http
/usr/local/Cellar/go/1.18/libexec/src/net/http/h2_bundle.go:7130:48: ErrDeadlineExceeded not declared by package os
make: *** [bin/tinygo-example.wasm] Error 1
$ PATH=/usr/local/go/bin:$PATH
$ go version
go version go1.17.6 darwin/amd64
$ make bin/tinygo-example.wasm
env GOROOT=/usr/local/go tinygo build -target=wasi -wasm-abi=generic -gc=leaking -scheduler=asyncify  -o bin/tinygo-example.wasm main.go
$
```

It's not actually using net/http for much.  Adding that error code to os lets the test program compile and run properly.  PR coming.

--- a/src/os/errors.go
+++ b/src/os/errors.go
@@ -19,6 +19,24 @@ var (
 )
 
 // The following code is copied from the official implementation.
+// src/internal/poll/fd.go
+
+// ErrDeadlineExceeded is returned for an expired deadline.
+// This is exported by the os package as os.ErrDeadlineExceeded.
+var ErrDeadlineExceeded error = &DeadlineExceededError{}
+
+// DeadlineExceededError is returned for an expired deadline.
+type DeadlineExceededError struct{}
+
+// Implement the net.Error interface.
+// The string is "i/o timeout" because that is what was returned
+// by earlier Go versions. Changing it may break programs that
+// match on error strings.
+func (e *DeadlineExceededError) Error() string   { return "i/o timeout" }
+func (e *DeadlineExceededError) Timeout() bool   { return true }
+func (e *DeadlineExceededError) Temporary() bool { return true }
+
+// The following code is copied from the official implementation.
 // https://github.com/golang/go/blob/4ce6a8e89668b87dce67e2f55802903d6eb9110a/src/os/error.go#L65-L104
 
 func NewSyscallError(syscall string, err error) error {

